addon/components/utils/address-form.ts

```diff
@@ -83,4 +125,50 @@ export default class extends Component<UtilsAddressFormArgs> {
       return isEmpty(get(this.args.address, addressAttr)) || shortAddress || invalidCountryCode || invalidZipcode;
     });
   }
+
+  private fillInAddress(place: any): void {
```

Comment:
Can't we have a proper type for the `place` parameter ?

Reply:
https://github.com/googlemaps/js-api-loader#typescript (worth testing)